### PR TITLE
Finalize Help Page

### PR DESCRIPTION
**Pull request by westlywright**
Added dynamic content to help page
Cleaned up styles
Add logic around new setting (help.enabled) to be able to turn off the help page. If help page is disabled but you've got no host we'll fall back to old welcome screen.

### CODE REVIEW

**Comment by westlywright** [comment]
rancher/rancher#3019

File: app/authenticated/help/controller.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+
+  modelResolved: false,
+  hasServices: true,
+  forumsLink: 'http://forums.rancher.com/',
```
Comment (vincent99) [inline app/authenticated/help/controller.js]: This should probably go in constants..

File: app/authenticated/help/route.js [hunk 2]
```diff
@@ -1,4 +1,47 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  actions: {
+    didTransition: function() {
+      Ember.$.getJSON('http://forums.rancher.com/categories.json').then((response) => {
```
Comment (vincent99) [inline app/authenticated/help/route.js]: ...and this re-use that constant

File: app/authenticated/help/template.hbs [hunk 3]
```diff
@@ -48,7 +58,7 @@
 </section>
 
 <div class="over-hr">
-  <span><a href="http://rancher.com/learn/">Getting Started</a> with Rancher</span>
+  <span><a href="http://rancher.com/learn/" target="_blank">Getting Started</a> with Rancher</span>
```
Comment (vincent99) [inline app/authenticated/help/template.hbs]: And all the ones in here too, at least the base URLs, like

company=http://rancher.com
github=https://github.com/rancher/rancher
docs=http://docs.rancher.com/rancher
forum=http://forums.rancher.com

**Comment by vincent99** [comment]
- Getting Started link in the header doesn't seem very visible..

- Alignment:
image

- Just under -lg:
image

- Small:
image

**Pull request merged**